frontend/frontendAngular/src/app/components/register-company/register-company.component.html

```diff
@@ -71,7 +71,7 @@
       </form>
     </mat-card>
     <div class="m-5">
-      <img src="assets/SVGCadastroCompany.svg" alt="SVGCadastroCompany">
+      <img src="assets/SVGRegisterCompany.svg" alt="SVGCadastroEmpresa">
```

Comment:
manter nomes em inglês no codigo mas o que for pro publico deixar em pt
substituir svgcadastrocompany por svgregistercompany e o svgcadastrocompany por svgcadastroempresa